BigFileMergeSort/Arrays.java

```diff
@@ -0,0 +1,192 @@
+package ua.goit.alg;
+
+import java.io.DataOutputStream;
+import java.io.EOFException;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.RandomAccessFile;
+import java.util.ArrayList;
+import java.util.NoSuchElementException;
+import java.util.Scanner;
+
+public class Arrays {
+
+    public static File mergeSort(File file, int blocksize) throws IOException {
+	ArrayList<File> partList = splitSort(file, blocksize);
+	File result = new File("result.txt");
```

Comment:
ideally output file should be a parameter / temp file so that people can safely call this on multiple files simultaneously